index.js

```diff
@@ -258,7 +269,7 @@ let rcl = {
             reject(e);
           });
       } else {
-        throw new Error('The module rclnodejs has been initialized.');
+        resolve();
```

Comment:
If we used the existence of context, we could remove this `else` I think

Reply:
see my updated version.